ocaml/xapi/xapi_pif.ml

```diff
@@ -767,6 +767,19 @@ let set_property ~__context ~self ~name ~value =
         Nm.bring_pif_up ~__context pif
     ) (self :: vlan_pifs)
 
+let pif_has_clustering_enabled ~__context (self : API.ref_PIF) =
+  let network_ref = Db.PIF.get_network ~__context ~self |> Ref.string_of in
+  (Db.Cluster.get_refs_where ~__context
+     ~expr:Db_filter_types.(Eq(Literal network_ref,Field "network")))
```

Comment:
As discussed, maybe it's safer to put `Field` first and then `Literal` to follow the existing conventions.